docs/data/base/getting-started/quickstart/quickstart.md

```diff
@@ -38,3 +36,27 @@ Please note that [react](https://www.npmjs.com/package/react) and [react-dom](ht
   "react-dom": "^17.0.0 || ^18.0.0"
 },
 ```
+
+## Import a component
+
+```jsx
+import * as React from 'react';
+import * as NumberField from '@base_ui/react/NumberField';
```

Comment:
We used 
```suggestion
import * as BaseNumberField from '@base_ui/react/NumberField';
```

in the demos of the docs, maybe it's not relevant here.

Reply:
Which docs? The current demos use `import * as NumberField from '@base_ui/react/NumberField';` as far as I can see.